packages/react-storage/src/components/StorageBrowser/displayText/types.ts

```diff
@@ -22,6 +22,7 @@ export interface DefaultListViewDisplayText {
   searchPlaceholder: string;
   searchSubmitLabel: string;
   searchClearLabel: string;
+  getDateText: (date: Date) => string;
```

Comment:
`getDateLabel` for consistency?

Reply:
Per my other comment - maybe getDateDisplayText? Esp since it's not quite a label for a date either?

Reply:
Had thought abt suggesting `getDateValue` here but iirc we already have a `getDownloadLabel` key, would prefer we keep consistency with appending either `value` or `label` for these